app/resources/css-default.template

```diff
@@ -597,6 +597,11 @@ td.submit {
   margin: 1em 0;
 }
 
+.main .row.custom-message {
+  margin: 24px auto;
+  text-align: center;
```

Comment:
Can you remove this line as I wrote:

> Also, I feel text-align: center is not necessary as long as you use "auto" margin to centerize the box.

Reply:
@gimite I had tried removing it, but the centering goes away.

Reply:
Oh I see. Looks like you can fix the issue by:
* Add `display: table;` in this rule
* Remove `width: 90%;` from `.custom-message` rule (which I believe is fine)